### ISSUE TITLE
Add list of moved files to landing zone moving timeline event

### ISSUE THREAD

**Issue by mikkonie** [bihealth/sodar-server#1202]
Something I thought of while discussing orhan data with @mkuhring. This would help us in tracking which user uploaded which files in case of e.g. orphans.

Will write out a more detailed spec when I get to it. This should be very straightforward.

**Update**: This depends on #691, so postponing. See sodar_taskflow#84 for more info.

## Spec

- Grab `self.tl_event` from the flow
    * This is technically optional, so if it's not there simply skip the rest
- Add new task to `sodar_tasks` to update timeline event extra data
- Add task to `landing_zone_move` at the end before landing zone status update
- Set list of files in timeline event extra data

## Tasks

- [x] Create task (see #2105)
- [x] Update task tests to test for tasks (see #2015)
- [x] Add task in flow
- [ ] Update flow tests
- [ ] Document

**Comment by mikkonie** [comment]
Done. One downside of this is that timeline event extra data is only viewable by owners and delegates. In this case, it should IMHO be viewed by anyone with access to the project.

To mend this we first need to implement some kind of a timeline event permission override in SODAR Core. See bihealth/sodar-core#386.
